src/libraries/Microsoft.Extensions.FileProviders.Physical/pkg/Microsoft.Extensions.FileProviders.Physical.pkgproj

```diff
@@ -4,6 +4,12 @@
     <ProjectReference Include="..\src\Microsoft.Extensions.FileProviders.Physical.csproj">
       <SupportedFramework>net461;netcoreapp2.0;uap10.0.16299;$(AllXamarinFrameworks)</SupportedFramework>
     </ProjectReference>
+
+    <!-- 
+      Suppress NETStandard.Library collpasing as it add more dependencies then needed in some 
+      scenarios like .NET Framework which adds an unecessary amount of package dependencies to download 
+    -->
+    <SuppressMetaPackage Include="NETStandard.Library" />
```

Comment:
Why would we ever be adding this on .NETFramework?  Can you share the nuspec before this change?

Reply:
I removed this as it is not required to mitigate this as I was able to do so by simply adding a package reference directly on the implementation instead of referencing the one from the targeting pack to make sure the assembly version is downgraded but do note that NETStandard.Library dependency is being added to all Microsoft.Extensions packages on net461 which doesn't really make much sense to me, I can dig a bit deeper to figure out why is that.

Reply:
> NETStandard.Library dependency is being added to all Microsoft.Extensions packages on net461 

That is indeed confusing.  We don't want that to happen.

Reply:
Agreed, I'll log an issue to investagate and fix that after this is in as it is not a problem specific due to these changes

Reply:
Logged this to track that: https://github.com/dotnet/runtime/issues/39225